src/Goerli-DssSpell.sol

```diff
@@ -50,79 +52,119 @@ contract DssSpellAction is DssAction {
-
-    function _updateDoc(bytes32 ilk, string memory doc) internal {
-        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
-        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
-
-        // Init the RwaLiquidationOracle to reset the doc
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
-            ilk, // ilk to update
-            0,   // price ignored if init() has already been called
-            doc, // new legal document
-            tau  // old tau value
-        );
-    }
-
     function actions() public override {
-        // -------------------- Update RWA007, RWA008, RWA009 Legal Documents ---------------------
-        // https://forum.makerdao.com/t/nov-16-2022-executive-contents/18747
-        //
-        // Monetalis (RWA007-A) legal update doc
-        _updateDoc("RWA007-A", "QmejL1CKKN5vCwp9QD1gebnnAM2MJSt9XbF64uy4ptkJtR");
-        // SG Forge OFH (RWA008-A) legal update doc
-        _updateDoc("RWA008-A", "QmZ4heYjptvj3ovafADJpXYMFXMyY3yQjkTXpvjFPnAKcy");
-        // HVB (RWA009-A) legal update doc
-        _updateDoc("RWA009-A", "QmeRrbDF8MVPQfNe83gWf2qV48jApVigm1WyjEtDXCZ5rT");
-
-        // RWA007-A autoline changes:
-        // - bump oralce price to 500m
-        // - increase DC to 500m
-        // - increase autoline `gap` to 100m
-        //
-        // https://vote.makerdao.com/polling/QmSfMtTM#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RWA007-A",
-            500 * MILLION,
-            100 * MILLION,
-            1 weeks
-        );
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
-            "RWA007-A",
-             500 * MILLION * WAD
-        );
-        DssExecLib.updateCollateralPrice("RWA007-A");
-
-        // RETH-A autoline changes:
-        // - line 5m
-        // - gap 3m
-        // - ttl 8h
-        //
-        //  https://vote.makerdao.com/polling/QmfMswF2#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RETH-A",
-            5 * MILLION,
-            3 * MILLION,
-            8 hours
-        );
-
-        // -------------------- Changelog Update & housekeeping ---------------------
-
-        // - Change "RWA007_A_INPUT_CONDUIT_URN" to "RWA007_A_INPUT_CONDUIT"
-        // - Change "RWA007_A_INPUT_CONDUIT_JAR" to "RWA007_A_JAR_INPUT_CONDUIT"
-        //
-        DssExecLib.setChangelogAddress("RWA007_A_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_URN"));
-        DssExecLib.setChangelogAddress("RWA007_A_JAR_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_JAR"));
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_URN");
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_JAR");
-
-        // Bump version
-        DssExecLib.setChangelogVersion("1.14.5");
+        // ----------------- Offboard GUSD-A, USDC-A and USDP-A -----------------
```

Comment:
I would rephrase it as it seems to be a liquidation activation and not an offboarding

Reply:
++ on this, this confused me when I was using the checklist for a minute.